tracer/build/_build/Build.Steps.cs

```diff
@@ -1026,7 +1098,7 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
 
             // Allow restore here, otherwise things go wonky with runtime identifiers
             // in some target frameworks. No, I don't know why
-            DotnetBuild(regressionLibs, framework: Framework, noRestore: false);
```

Comment:
Going to stop marking these, but we don't want to remove the framework condition, otherwise we're doing more work than we want to

Reply:
Oops, this was left over from a little clean-up I was trying. I'll revert all the unrelated changes 👍🏽 

(although, this parameter seems to always have the value of `Framework`, so it seems redundant)

Reply:
> although, this parameter seems to always have the value of Framework, so it seems redundant to pass it around

Well, except where its not specified, which is in quite a lot of places 😄